CHANGES.md

```diff
@@ -1,5 +1,153 @@
 # Changes
 
+## stable-2.13.0
+
+This release introduces client-side policy to Linkerd, including dynamic routing
+and circuit breaking. [Gateway API](https://gateway-api.sigs.k8s.io/) HTTPRoutes
+can now be used to configure policy for outbound (client) proxies as well as
+inbound (server) proxies, by creating HTTPRoutes with Service resources as their
+`parentRef`. See the Linkerd documentation for tutorials on [dynamic request
+routing] and [circuit breaking]. New functionality for debugging HTTPRoute-based
+policy is also included in this release, including [new proxy metrics] and a new
+`linkerd diagnostics policy` CLI command.
+
+In addition, this release adds `network-validator`, a new init container to be
+used when CNI is enabled. `network-validator` ensures that local iptables rules
+are working as expected. It will validate this before linkerd-proxy starts.
+`network-validator` replaces the `noop` container, runs as `nobody`, and drops
+all capabilities before starting.
+
+Finally, this release includes a number of bugfixes, performance improvements,
+and other smaller additions.
+
+**Upgrade notes**: Please see the [upgrade instructions][upgrade-2130].
+
+* CRDs
+  * HTTPRoutes may now have Service parents, to configure outbound policy
+  * Updated HTTPRoute version from `v1alpha1` to `v1beta2`
+
+* CLI
+  * Added a new `linkerd prune` command to the CLI (including extensions) to
+    remove resources which are no longer part of Linkerd's manifests
+  * Added additional shortnames for Linkerd policy resources (thanks @javaducky!)
+  * The `linkerd diagnostics policy` command now displays outbound policy when
+    the target resource is a Service
+
+* Destination Controller
+  * Lowered non-actionable error messages in the Destination log to debug-level
+    entries to avoid triggering false alarms (thanks @siddharthshubhampal!)
+  * Fixed an issue with EndpointSlice endpoint reconciliation on slice deletion;
+    when using more than one slice, a `NoEndpoints` event would be sent to the
+    proxy regardless of the amount of endpoints that were still available
+    (thanks @utay!)
+  * Improved diagnostic log messages
+  * Fixed sending of spurious profile updates
+  * Removed unnecessary Namespaces access from the destination controller RBAC
+  * Added Kubernetes metadata API in the destination controller for watching Nodes
```

Comment:
This one's probably too low level to surface

Reply:
```suggestion
```